app/src/main/java/ch/epfl/skysync/viewmodel/LocationViewModel.kt

```diff
@@ -205,10 +204,9 @@ class LocationViewModel(uid: String? = null, val repository: Repository) : ViewM
     // do not take deletions into account, as it is just deletions from the query results
     // that is: it is not the location with the highest time anymore
     val updatedLocations = update.adds + update.updates
-    locations =

```

Comment:
Please do not modify `LocationViewModel` (unless it is broken ?), I'm currently working on it (#321) and it'll just result in avoidable merge conflicts.